### PR TITLE
cnv 2.5: VMware import supports Ceph block-format storage

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1874786

CP 4.5, 4.6

### CODE REVIEW

File: modules/virt-importing-vm-wizard.adoc [hunk 1]
```diff
@@ -129,6 +130,10 @@ _or_
 
 *** VM disk: *NFS*
 *** `v2v-conversion-template` disk: *hostpath-provisioner (HPP)*
+endif::[]
+ifeval::["{VirtVersion}" >= "2.5"]
+** *Storage Class*: Select
```
Comment (bergerhoffer) [inline modules/virt-importing-vm-wizard.adoc:135]: This looks like it's supposed to be a sentence that's just not finished.

**Comment by apinnick** [comment]
@bergerhoffer Please merge. Thanks.

**Pull request merged**